shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/server/plugins/PluginManager.java

```diff
@@ -1889,7 +1889,8 @@ public static CallSite bootstrapServices(
               collector,
               callsite.dynamicInvoker().asType(MethodType.methodType(RequiredServices.class)));
     }
-    // Now, we should have a method thats () → String[], which is what we wanted, so shove it in the
+    // Now, we should have a method that's () → String[], which is what we wanted, so shove it in
+    // the
```

Comment:
```suggestion
    // the olive
```
and delete line below?